apps/tufctl/main.cpp

```diff
@@ -0,0 +1,106 @@
+#include <boost/property_tree/ini_parser.hpp>
```

Comment:
This seems like its more for debug/development? I'm not sure what the long term point of this is - is more of an "examples"?

Reply:
Yes, this is rather a test and an example utility. That's why it's not built by default.

Reply:
y'all put a lot of thought into this so i won't bikeshed. But it seems like we might be developing an "examples" directory over time.